index.js

```diff
@@ -4,19 +4,24 @@
  Shamelessly based on bundle-loader by Tobias Koppers @sokra
  */
 
+ var path = require('path');
+
 module.exports = function () {};
 module.exports.pitch = function (remainingRequest) {
   this.cacheable && this.cacheable();
   var query = this.query.substring(1).split(','),
     promiseLib = query[0],
     bundleName = query[1] || '';
+  var file = path.parse(remainingRequest);

```

Comment:
AFAIK `path.parse` isn't available in 0.10 which some people still use. Better to use `path.basename` instead.